app/src/main/java/com/eveningoutpost/dexdrip/Home.java

```diff
@@ -2480,6 +2483,9 @@ public void onClick(DialogInterface dialog, int which) {
         if(DexCollectionType.isLibreOOPAlgorithm(collector)) {
         	// Rest of this function deals with initial calibration. Since we currently don't have a way to calibrate,
         	// And even once we will have, there is probably no need to force a calibration at start of sensor use.
+            displayCurrentInfo();
+            // JamorHam, should I put here something like:
+            // ?? if (screen_forced_on)  dontKeepScreenOn();
```

Comment:
Please see if I need to add something like:
if (screen_forced_on)  dontKeepScreenOn();